handlers/anime.py

```diff
@@ -111,7 +111,13 @@ def __init__(self):
         if message.content.startswith(self.signal):
 
             msg = message.content.replace(self.signal+" ", "")
-            top, bottom = msg.split("|")
+
+            if "|" in msg:
+                top, bottom = msg.split("|")
+            else:
+                top = " "
+                bottom = " "
```

Comment:
Consider setting one of these (probably bottom) to the message, if it exists, so content is not lost

Reply:
The intent of not handling cases like this was so that you could specify if you wanted the text on the top or bottom via the pipe character.
e.g.
`text on top|`
`|text on bottom`
`text on|both`

Reply:
This change would not remove the functionality of `top|` or `|bottom` or `both|both`, as those would be handled by the if statement. This would just handle the case where no pipe character is found and still try to use it. It will add less-defined functionality, but I believe it's intuitive enough UX-wise.

Reply:
Robbot was throwing errors when a pipe symbol wasn't included. Not crashing, just throwing errors. This is setting a default for when the pipe isn't included and to prevent throwing errors.